src/Engine/Main.py

```diff
@@ -0,0 +1,24 @@
+from src.Display.ReadSmoothieFile import Menu
+from src.Entities.CustomerOrder import CustomerOrder
+
+
+def main():
+    combine = ""
+    menu = Menu()
+    customerMenu = menu.Smoothie_Menu()
+
+    for words in customerMenu:
+        combine += "\n" + ", ".join(words
+                                    )
```

Comment:
I would fix the formatting of this as the parenthesis doesn't need to be on a new line